pavement.py

```diff
@@ -28,7 +28,12 @@
 #
 # =================================================================
 
-from ConfigParser import SafeConfigParser
+from __future__ import (absolute_import, division, print_function)
+
+try:
```

Comment:
Tried that and:
```python
  File "pavement.py", line 33, in <module>
    from six.moves.configparser import SafeConfigParser
```
Which I'm **guessing** is related to something in Paver..  Keeping existing unless there's a better way?